openpype/plugins/publish/validate_version.py

```diff
@@ -25,16 +25,16 @@ def process(self, instance):
             # TODO: Remove full non-html version upon drop of old publisher
             msg = (
                 "Version '{0}' from instance '{1}' that you are "
-                " trying to publish is lower or equal to an existing version "
-                " in the database. Version in database: '{2}'."
+                "trying to publish is lower or equal to an existing version "
+                "in the database. Version in database: '{2}'."
```

Comment:
Haha, @kalisp it's only AFTER this is merged of course that I notice that this line should have a space after the dot at the end. :(

Reply:
Btw I myself would use `chore` label in branch as this is not much of an `enhancement`, but just house cleaning. But that just me. 
I think that `feature` and `enhancement` goes to Release notes, I don't think this needs to.